tests/serialization/test_parameter_serialization.py

```diff
@@ -9,25 +9,25 @@
 
 from ..conftest import _CHEM_INSTALLED
 from ..hypothesis_strategies.parameters import (
-    categorical_parameter,
-    custom_parameter,
-    numerical_continuous_parameter,
-    numerical_discrete_parameter,
-    substance_parameter,
-    task_parameter,
+    categorical_parameters,
```

Comment:
Just to be extra annoying (pls dont get angry 😢 ): This file would not have been part of the PR if it was only dealing with Hypothesis for objective 😜

Reply:
Yes, so actually same issue as mentioned in the other thread. Will close this one here so we have only one point of discussion